pkg/decode/decode.go

```diff
@@ -688,11 +688,15 @@ func (d *D) trySeekAbs(pos int64, fns ...func(d *D)) (int64, error) {
 	return pos, nil
 }
 
-// SeekRel seeks relative to current bit position
+// TrySeekRel seeks relative to current bit position. If one or more functions is
+// passed, the original seek position will be restored after decoding.
```

Comment:
👍

Reply:
Would be nice if this could be limited to just one optional function but no possible to express in go